docsite/docs/api_pages/object-detection.md

```diff
@@ -10,4 +10,424 @@ For more details or to demo it, visit
+     setCategories(
+       results.results
+         .map(result =>
+           result.detections
+             .map(d => d.categories.map(c => c.categoryName).join(', '))
+             .join(', '),
+         )
+         .join(', '),
+     );
+   },
+   error => {
+     console.error(`onError: ${error}`);
+   },
+   RunningMode.LIVE_STREAM,
+   'efficientdet-lite0.tflite',
+   {delegate: Delegate.GPU},
+ );
+
+
+ return (
+   <SafeAreaView style={styles.root}>
+     {cameraPermission.hasPermission ? (
+       <View style={styles.container}>
+         <MediapipeCamera style={styles.camera} processor={frameProcessor} />
+         <Text style={styles.categoriesText}>{categories}</Text>
+       </View>
+     ) : (
+       <RequestPermissions
+         hasCameraPermission={cameraPermission.hasPermission}
+         requestCameraPermission={cameraPermission.requestPermission}
+       />
+     )}
+   </SafeAreaView>
+ );
+}
+
+
+const RequestPermissions: React.FC<{
+ hasCameraPermission: boolean;
+ requestCameraPermission: () => Promise<boolean>;
+}> = ({hasCameraPermission, requestCameraPermission}) => {
+ console.log(hasCameraPermission);
+ return (
+   <View style={styles.container}>
+     <Text style={styles.welcome}>Welcome to React Native Mediapipe</Text>
+     <View style={styles.permissionsContainer}>
+       {!hasCameraPermission && (
+         <Text style={styles.permissionText}>
+           React Native Mediapipe needs{' '}
+           <Text style={styles.bold}>Camera permission</Text>.{' '}
+           <Text style={styles.hyperlink} onPress={requestCameraPermission}>
+             Grant
+           </Text>
+         </Text>
+       )}
+     </View>
+   </View>
+ );
+};
+
+
+const styles = StyleSheet.create({
+ root: {
+   flex: 1,
+   justifyContent: 'center',
+   alignItems: 'center',
+   color: 'black',
+ },
+ welcome: {color: 'black', fontSize: 38, fontWeight: 'bold', maxWidth: '80%'},
+ banner: {
+   position: 'absolute',
+   opacity: 0.4,
+   bottom: 0,
+   left: 0,
+ },
+ container: {
+   height: '100%',
+   width: '100%',
+   backgroundColor: 'white',
+   flexDirection: 'column',
+ },
+ camera: {
+   flex: 1,
+ },
+ categoriesText: {color: 'black', fontSize: 36},
+ permissionsContainer: {
+   marginTop: 30,
+ },
+ permissionText: {
+   color: 'black',
+   fontSize: 17,
+ },
+ hyperlink: {
+   color: '#007aff',
+   fontWeight: 'bold',
+ },
+ bold: {
+   fontWeight: 'bold',
+ },
+});
+
+
+export default App;
+```
+
+6. **Reaload app**: and grant acces to camera permissions
+  </TabItem>
+
+  <TabItem value="iOS" label="iOS">
+1. **Give Permissions:** Navigate to your info.plist file in the outermost tag:
+
+```jsx
+// add this at the end of your file before the closing </dict> tag
+
+  <key>NSCameraUsageDescription</key>
+  <string>$(PRODUCT_NAME) needs access to your Camera.</string>
+```
+
+
+2. **Terminal Commands:** In your terminal run the following commands
+
+:::info
+You will only need to run the bundle install command once.
```

Comment:
how often do you have to run pod install ?

Reply:
Like almost every time I have an error and can't identify it. It's usually a dependency one and it gets fixed by doing a pod install